mbuild/compound.py

```diff
@@ -1154,8 +1186,267 @@ def _kick(self):
+            Path to forcefield xml file
+        steps : int, optional, default=1000
+            Number of energy minimization iterations
+        scale_bonds : float, optional, default=1
+            Scales the bond force constant (1 is completely on)
+        scale_angles : float, optiona, default=1
+            Scales the angle force constant (1 is completely on)
+        scale_torsions : float, optional, default=1
+            Scales the torsional force constants (1 is completely on)
+        scale_nonbonded : float, optional, default=1
+            Scales epsilon (1 is completely on)
+
+
+        Notes
+        -----
+        Assumes a particular organization for the force groups
+        (HarmonicBondForce, HarmonicAngleForce, RBTorsionForce, NonBondedForce)
+
+        References
+        ----------
+
+        .. [1] P. Eastman, M. S. Friedrichs, J. D. Chodera, R. J. Radmer,
+               C. M. Bruns, J. P. Ku, K. A. Beauchamp, T. J. Lane,
+               L.-P. Wang, D. Shukla, T. Tye, M. Houston, T. Stich,
+               C. Klein, M. R. Shirts, and V. S. Pande.
+               "OpenMM 4: A Reusable, Extensible, Hardware Independent
+               Library for High Performance Molecular Simulation."
+               J. Chem. Theor. Comput. 9(1): 461-469. (2013).
+
+
+
+        """
+        to_parmed = self.to_parmed()
+        ff = Forcefield(forcefield_files=[forcefield])
+        to_parmed = ff.apply(to_parmed)
+
+        from simtk.openmm.app.simulation import Simulation
+        from simtk.openmm.app.pdbreporter import PDBReporter
+        from simtk.openmm.openmm import LangevinIntegrator
+        import simtk.unit as u
+
+        system = to_parmed.createSystem()
+        integrator = LangevinIntegrator(298 * u.kelvin, 1 / u.picosecond,
+                                        0.002 * u.picoseconds)
+        simulation = Simulation(to_parmed.topology, system, integrator)
+
+        for force in system.getForces():
+            if type(force).__name__ == "HarmonicBondForce":
+                for bond_index in range(force.getNumBonds()):
+                    atom1, atom2, r0, k = force.getBondParameters(bond_index)
+                    force.setBondParameters(bond_index,
+                                            atom1, atom2,
+                                            r0, k * scale_bonds)
+                force.updateParametersInContext(simulation.context)
+
+            elif type(force).__name__ == "HarmonicAngleForce":
+                for angle_index in range(force.getNumAngles()):
+                    atom1, atom2, atom3, r0, k = force.getAngleParameters(
+                        angle_index)
+                    force.setAngleParameters(angle_index,
+                                             atom1, atom2, atom3,
+                                             r0, k * scale_angles)
+                force.updateParametersInContext(simulation.context)
+
+            elif type(force).__name__ == "RBTorsionForce":
```

Comment:
Should probably specify in the docstring that only Rykaert-Belleman's style torsions are supported as of now. We could expand support to other functional forms, but let's go ahead and just document this limitation for now so that we can get this merged.